src/Commands.rb

```diff
@@ -2095,6 +2095,34 @@ def event_look_at?(ev, to, scope, metric = :square)
       end
       return x_axis && y_axis && (distance_between(metric, ev, to)<=scope)
     end
+    def event_look_towards_event?(ev1, ev2, scope)
+      event_look_towards?(ev1, event_x(ev2), event_y(ev2), scope)
+    end
+    def event_look_towards?(ev, x, y, scope)
+      ex, ey = event_x(ev), event_y(ev)
+      if event_direction(ev) == 8
+        distance = ey - y
+        x_axis = ex >= x - distance && ex <= x + distance
```

Comment:
### [4] Variables' naming

I would rather use `is_in_x_axis` and `is_in_y_axis` instead of `x_axis` and `y_axis` as they are storing boolean values stating whether the *event* is in the `x` or `y` axis of the observer's sight.

### [5] Idiomatic structure

These instructions are meant to check if a value (`x`) is contained within a given range.
Thus, wouldn't it be easier-to-read using `(min..max).include? x` ?
```ruby
x_axis = (x - distance .. x + distance).include? ex
```

(Or at least put some parentheses to quickly understand operations' priorities !<br/>&nbsp;For instance: `x_axis = (x - distance <= ex) && (ex <= x + distance)`)

Reply:
[4] => I don't really like the form is_... for variables or function/method naming. As it is something used just below, I think the names x_axis and y_axis are enough.

[5] => Not ok for changing into a range. Beside losing in perf (making a range and then searching into it) I don't see much change, it will still be as lisible as two comparaisons.
But I agree with the parenthesis, I'll put them there.